pgjdbc/src/main/java/org/postgresql/util/PGInterval.java

```diff
@@ -240,12 +240,16 @@ public void setValue(int years, int months, int days, int hours, int minutes, do
    * @return String represented interval
    */
   public String getValue() {
-    return years + " years "
-        + months + " mons "
-        + days + " days "
-        + hours + " hours "
-        + minutes + " mins "
-        + wholeSeconds + '.' + microSeconds + " secs";
+    return String.format(
+      Locale.ROOT,
+      "%d years %d mons %d days %d hours %d mins %f secs",
+      years,
+      months,
+      days,
+      hours,
+      minutes,
+      wholeSeconds + microSeconds / 1e6d
```

Comment:
I've started using TimeUnit.* to convert to and from time units. It's a bit more readable.